src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+@import "src/styles/utils/variables";
+
+.card-wrp {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  width: $width-wrp;
+
+  margin: auto;
+  padding: 50px 0;
+}
+
+.card {
+  width: $card-width;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  overflow: hidden;
+
+  padding: 0 16px 16px;
+
+  &:not(:nth-child(#{$cards-in-row}n)) {
+    margin-right: $card-space;
+  }
+
+  &:not(:nth-last-child(-n + #{$cards-in-row})) {
+    margin-bottom: 46px;
+  }
```

Comment:
Use BEM. Don't use external styles in .card selector. Add .card-wrp__card selector on card.
and better rename .card-wrp to .catalog or smth else